app/main.py

```diff
@@ -1,8 +1,24 @@
 class Person:
-    # write your code here
-    pass
 
+    people = {}
 
-def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
+
+
+def create_person_list(people_list: list[dict[str]]) -> list:
+    new_person_list = [Person(person.get("name"),
+                              person.get("age")) for person in people_list]
+
+    for person in people_list:
+        if person.get("wife"):
+            wife = Person.people[person.get("wife")]
+            Person.people[person.get("name")].wife = wife
+
+        if person.get("husband"):
+            husband = Person.people[person.get("husband")]
+            Person.people[person.get("name")].husband = husband
```

Comment:
Similar to the 'wife' key, the code assumes that the 'husband' key refers to an existing `Person` object in `Person.people`. If the name does not exist, this will raise a `KeyError`. Add a check to ensure the name exists in `Person.people` before accessing it.